src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -164,5 +170,32 @@ internal static Settings FromDefaultSources()
 
             return new Settings(configurationSource);
         }
+
+        private static OtlpExportProtocol GetExporterOtlpProtocol(IConfigurationSource source)
+        {
+            // the defualt in SDK is grpc. It is oposite that here.
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
+
+            if (exporterOtlpProtocol == "grpc")
+            {
+                return OtlpExportProtocol.Grpc;
+            }
+
+            return OtlpExportProtocol.HttpProtobuf;
```

Comment:
what do you think of changing to something more or less like:

```csharp
var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
if string.IsNullOrEmpty(exporterOtlpProtocol)
{
    exporterOtlpProtocol  = "http/protobuf";
}
switch (exporterOtlpProtocol) {
case "http/protobuf":
   return OtlpExportProtocol.HttpProtobuf;
case "grpc":
   return tlpExportProtocol.Grpc;
default:
   throw new Exception(blalbla);
```

the problem is that the current code does not handle invalid values -> e.g. when one sets `blabla` as the env var

the other possibility could be doing something like

```csharp
var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
if string.IsNullOrEmpty(exporterOtlpProtocol)
{
    return OtlpExportProtocol.HttpProtobuf; // override the default
}
return null; // keep the existing value, do not override anything
```

then the exporter setup code would have to be changed to:

```csharp
builder.AddOtlpExporter(options =>
{
    if (settings.OtlpExportEndpoint != null) options.Protocol = settings.OtlpExportProtocol;
    // ...
});
```

I like the second approach more as it does not require parsing the values and it would not require any changes if support for `http/json` protocol will be added in the SDK